x-pack/plugins/security_solution/public/common/components/exceptions/helpers.tsx

```diff
@@ -81,39 +78,10 @@ export const getExceptionOperatorSelect = (item: BuilderEntry): OperatorOption =
 };
 
 /**
- * Formats ExceptionItem entries into simple field, operator, value
- * for use in rendering items in table
+ * Returns the fields corresponding value for an entry
  *
- * @param entries an ExceptionItem's entries
+ * @param item a single ExceptionItem entry
  */
-export const getFormattedEntries = (entries: BuilderEntry[]): FormattedEntry[] => {
```

Comment:
These big chunks that appear as removed were just moved into `builder/helper.ts` since they weren't shared utils, just applied to the builder. Was trying to cleanup as this file is getting large.